apps/picsa-tools/option-tool/src/app/pages/home/home.component.ts

```diff
@@ -25,15 +23,28 @@ const COLUMNS = [
   },
 ];
 export interface IOptionData {
-  name: string;
-  position: number;
-  weight: number;
-  symbol: string;
+  practice: string;
+  gender: string [];
+  benefits: {benefit:string, beneficiary:string[]} [];
```

Comment:
(+)
Appreciate you thinking about these data structures, and that fact that there can be multiple beneficiaries to each benefit

Reply:
thanks